ere-guests/stateless-validator/sp1/src/main.rs

```diff
@@ -46,3 +47,38 @@ fn init_tracing_just_like_println() {
         .with_max_level(tracing::Level::INFO) // capture info! and up
         .init(); // set as global default
 }
+
+fn from(fork_spec: ForkSpec) -> ChainSpec {
```

Comment:
This would be the alternative `.into()` logic that we could upstream [here](https://github.com/kevaundray/reth/blob/75092d6aa098311db96c06b97a252a6f2b42dbaf/crates/stateless/src/fork_spec.rs#L63). For now leaving it here to avoid forking deps.